docs/content/rules/sort-array-includes.mdx

```diff
@@ -200,6 +205,7 @@ Allows you to use comments to separate the members of arrays into logical groups
 -	`false` — Comments will not be used as delimiters.
 - `string` — A regexp pattern to specify which comments should act as delimiters.
 - `string[]` — A list of regexp patterns to specify which comments should act as delimiters.
+- `{ block: BaseType; line: BaseType }` — Specify which block and line comments should act as delimiters.
```

Comment:
I'm not sure it's clear what BaseType means here.

Reply:
@azat-io The idea is to tell that `block` and `line` can be `string | string[] | boolean`.

I felt that 
```ts
`{ block: string | string[] | boolean; line: string | string[] | boolean }`
```

was a bit hard to read but we can do that if you feel that it's better.